Source/Plugins/CrossPlatformPluginGenerator/Assets/Microsoft.SpatialAudio.Spatializer.Unity/Plugins/WSA/ARM/AudioPluginMicrosoftSpatializerCrossPlatform.pdb.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
I wonder why we place PDBs in the Unity directories with the binaries?  Seems like we don't need to ship PDBs in anything other than the artifact feed and leave it out of the Unity package.

Reply:
We [intentionally did a change to ship PDBs](https://github.com/microsoft/spatialaudio-unity/commit/d76600c47832ae35214cf67159ee6e6d7bf072b7) so customers can post meaningful stacks when they hit issues. They don't have access to the artifacts feed, just UPM feed and the unitypackage on the releases tab.